PSKoans/Koans/Cmdlets 2/AboutOutCmdlets.Koans.ps1

```diff
@@ -115,23 +115,20 @@ Context 'Out-* Cmdlets' {
             Out-String is a fantastic and simple way to turn complex objects into strings that look just
             like PowerShell's default console formatting.
         #>
-
+    
         It 'creates string representations of data' {
-            $HomeFolder = Get-Item -Path $HOME
-
-            $HomeFolder | Should -Not -BeNullOrEmpty
-            $String = $HomeFolder | Out-String
-
-            # Fill in this here-string with the data you'd see in your console from calling Get-Item $HOME
+            #Create a hashtable that pipes into Out-String
+            $String = '__' | Out-String
+    
             @"
-
-
-    Directory: __
-
-
-Mode                LastWriteTime         Length Name
-----                -------------         ------ ----
-______         __________________                ____
+    
+    Name                           Value                                                                                                                                                                                         
+    ----                           -----                                                                                                                                                                                         
+    ColumnA                        Value1                                                                                                                                                                                        
+    ColumnB                        Value2                                                                                                                                                                                        
+    
+    
+    
```

Comment:
I think there's one too many newlines here compared to what will typically be returned from Out-String; this will probably be enough to fail the test.

Reply:
Looks like VSCode helped me out with some formatting. :)